todo-app/src/state.js

```diff
@@ -1,11 +1,36 @@
 import React from 'react'
 
-export default class State extends React.Component{
+export default class Status extends React.Component{
+    constructor(props) {
+        super(props);
+        this.state = {
+          done: this.props.status
+        }
+        this.changedStatus = this.changedStatus.bind(this);
+      }
+      changedStatus(){
+          let changed = !this.state.done
+          this.setState({
+              done: changed
+         })
+
+        }
+      getStatus() {
+          let undone = "Undone"
+          if (this.state.done) {
+              undone = "done"
+          } return undone
+      }
     render(){
         return (
-        <div>
-            {this.props.state}
-        </div>
+            <div>
+                <input type="checkbox">
+                {/* checked={this.state.checked} */}
```

Comment:
You should remove `{/* checked={this.state.checked} */}` comment to make code easier to read. Also `input` should close like `<input type="checkbox" />`. You don't need `</input>`